sdk/core/core-client-rest/src/getClient.ts

```diff
@@ -162,13 +162,14 @@ function buildOperation(
   allowInsecureConnection?: boolean,
   httpClient?: HttpClient
 ): StreamableMethod {
+  allowInsecureConnection = options.allowInsecureConnection ?? allowInsecureConnection;
```

Comment:
@xirzec Does that mean we would use the operation-level one to override the client-level one?

If yes, would this case be an issue?

1. No client-level value set
2. The `allowInsecureConnection` is set as `true` in Op1
_3. So the client-level value is overrode as `true`(Not sure, is it right?)_
4. No operation-level value is set in Op2
5. But finally Op2 would accept the value as `true`